go/vt/vttest/local_cluster.go

```diff
@@ -237,6 +245,16 @@ func (db *LocalCluster) Setup() error {
 
 	log.Infof("LocalCluster environment: %+v", db.Env)
 
+	// Set up topo manager if we are using a remote topo server
+	if db.RemoteTopoImplementation != "" {
+		db.topo = db.Env.TopoManager(db.RemoteTopoImplementation, db.RemoteTopoGlobalServerAddress, db.RemoteTopoGlobalRoot, db.Topology)
+		log.Infof("Initializing Topo Manager (%T)...", db.topo)
+		if err := db.topo.Setup(); err != nil {
+			log.Errorf("Failed to set up Topo Manager: %s", err)
```

Comment:
Nit, but I think it's better to use the %v formatter with err or with %s use `err.Error()`

Reply:
+1